src/utils/baseClass.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import shallowCompare from 'react-addons-shallow-compare';
+
+export const pureComponentAvailable = typeof React.PureComponent !== 'undefined';
+
+export default function baseClass(pureComponent = pureComponentAvailable) {
```

Comment:
why is `baseClass` a function, but `pureComponentAvailable` is not?

I'd expect either both to be a function (to support polyfilling of PureComponent, i assume), or neither.